fluss-server/src/main/java/com/alibaba/fluss/server/log/WriterStateManager.java

```diff
@@ -419,6 +414,10 @@ private boolean isWriterExpired(long currentTimeMs, WriterStateEntry writerState
         return currentTimeMs - writerStateEntry.lastBatchTimestamp() > writerExpirationMs;
     }
 
+    public boolean isBatchExpired(long currentTimeMs, LogRecordBatch recordBatch) {
```

Comment:
How about renaming to `isWriterInBatchExpired` which I think is clearer?  `isBatchExpired` sounds like the batch is expired, but we don't have this concept.